src/main/java/org/snomed/aag/data/domain/CriteriaItem.java

```diff
@@ -48,6 +50,9 @@
 	@Field(type = FieldType.Keyword)
 	private String requiredRole;
 
+	@Field(type = FieldType.Keyword)
+	private Set<String> flags = new HashSet<>();
```

Comment:
FRI-182 has only mentioned a `complex` flag, but I've written the code to be flag-agnostic.

Reply:
Could you name this field something more meaningful please? The CriteriaItem doesn't really own the flag, it's activated by it. Property name could be `enabledByFlag` - I would name this as singular even those it's a Set because any of them would activate it.

Reply:
Thanks Kai, I have done this.